src/test/resources/data/bsearch2.json

```diff
@@ -0,0 +1,228 @@
+                      }
+                    }
+                  }
+                ],
+                "should":[
+
+                ],
+                "must_not":[
+
+                ]
+              }
+            }
+          },
+          "track_total_hits":false,
+          "preference":1617730961756
+        }
+      },
+      "options":{
+        "sessionId":"3e0ee321-2b66-4da9-a956-23a9fbf07289",
+        "isRestore":false,
+        "legacyHitsTotal":false,
+        "strategy":"ese",
+        "isStored":false
+      }
+    },
+    {
+      "request":{
+        "params":{
+          "index":"kibana_sample_data_ecommerce",
+          "body":{
+            "version":true,
+            "size":500,
+            "sort":[
+              {
+                "order_date":{
+                  "order":"desc",
+                  "unmapped_type":"boolean"
+                }
+              }
+            ],
+            "fields":[
+              {
+                "field":"*",
+                "include_unmapped":"true"
+              },
+              {
+                "field":"customer_birth_date",
+                "format":"strict_date_optional_time"
+              },
+              {
+                "field":"order_date",
+                "format":"strict_date_optional_time"
+              },
+              {
+                "field":"products.created_on",
+                "format":"strict_date_optional_time"
+              }
+            ],
+            "script_fields":{
+
+            },
+            "stored_fields":[
+              "*"
+            ],
+            "runtime_mappings":{
+
+            },
+            "_source":false,
+            "query":{
+              "bool":{
+                "must":[
+
+                ],
+                "filter":[
+                  {
+                    "match_all":{
+
+                    }
+                  },
+                  {
+                    "match_all":{
+
+                    }
+                  },
+                  {
+                    "range":{
+                      "order_date":{
+                        "gte":"2021-03-30T17:42:53.295Z",
+                        "lte":"2021-04-06T17:42:53.295Z",
+                        "format":"strict_date_optional_time"
+                      }
+                    }
+                  }
+                ],
+                "should":[
+
+                ],
+                "must_not":[
+
+                ]
+              }
+            },
+            "highlight":{
+              "pre_tags":[
+                "@kibana-highlighted-field@"
+              ],
+              "post_tags":[
+                "@/kibana-highlighted-field@"
+              ],
+              "fields":{
+                "*":{
+
+                }
+              },
+              "fragment_size":2147483647
+            }
+          },
+          "preference":1617730961756
+        }
+      },
+      "options":{
+        "sessionId":"3e0ee321-2b66-4da9-a956-23a9fbf07289",
```

Comment:
Can we update this parameter to be randomly generated for each search request? That will help keep things more realistic. Here's what I used in my load tests:

```scala
.replaceAll(
        "\"sessionId\":\"3e0ee321-2b66-4da9-a956-23a9fbf07289\"",
        s"""\"sessionId\": \"${Random.alphanumeric.take(10).mkString("")}\""""
      )
  }
```

(You'll have to use the same `sessionId` for a single load of all dashboard panels, so I imagine you'd reuse the randomly generated one for each of the places below where you have this same sessionId.)

Reply:
Definitely! Thank you for the idea.